index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav id="navbar">
+      <a href="#profile">profile</a>
+      <a href="#about">about me</a>
+      <a href="#contact">contact</a>
+    </nav>
+    <br />
+    <br />
+    <section id="profile">
+      <h1 id="name">
+        Tamana Balkhi <br />
+        software developer
+      </h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or

```

Comment:
- [ ] Please as the last reviewer said make sure to follow Figma template with your design 

| Your design | Figma design | 
| ---------------- | ------------------ |
| ![image](https://user-images.githubusercontent.com/74744787/176976920-414ad93e-4376-4f74-8a6a-eb77d92b886e.png) | ![image](https://user-images.githubusercontent.com/74744787/176977076-cbf0d3a5-c770-442b-929d-4d1dc93ffc8c.png) |